### DISCUSSION TITLE
This framework also works in Rider on all OS

### DISCUSSION THREAD

**Discussion by van800** [MikeSchulze/gdUnit4Mono#60] [Show and tell]
Awesome work with this framework.
I made a plan to polish the integration with Rider. https://github.com/JetBrains/godot-support/issues/137
It mostly works, but need to fix couple of things and pass the GODOT_BIN somehow automatically, since Rider already knows the path to Godot.

**Comment by MikeSchulze** [comment]
Very nice! Thanks for your work to get the test adapter running with Rider.
The project needs to build on Godot/SDK based because of the assert supports Godot and standard C# objects.
Also needs to be able to load the test suites inside the Godot Editor to run C# test from the GdUnit4 inspector.

**Comment by van800** [comment]
I have just published a version of the Godot Support plugin (2024.1.157), which supports gdUnit4Mono. There is still a need to enable VSTest in the Rider settings, I might address it later.
